lib/widgets/interactive/text_field.dart

```diff
@@ -282,6 +285,7 @@ class TwTextField extends TextField {
     this.placeholder,
     this.errorLabel,
     this.counter,
+    this.suffixIcon,
```

Comment:
this is a pass-through property so it should probably be placed at the bottom

Reply:
hey kevin, sorry this is so late but I tried setting the suffixicon as a passthrough parameter, but i realized that suffixicon isn't explicitly a parameter for Textfield, it's really a parameter to pass to `InputDeocration`. so having `super.suffixicon` in TwTextfield actually breaks things. here's an example usage:

```
Widget build(BuildContext context) {
    return const TextField(
      decoration: InputDecoration(
        border: OutlineInputBorder(),
        labelText: 'Enter password',
        suffixIcon: Align(
          widthFactor: 1.0,
          heightFactor: 1.0,
          child: Icon(
            Icons.remove_red_eye,
          ),
        ),
      ),
    );

Reply:
ah gotcha, that makes sense